gprofiler/profilers/__init__.py

```diff
@@ -0,0 +1,7 @@
+# flake8: noqa
+# TODO: Make sure to import any new process filters to load it
```

Comment:
"process filters"? "profilers"?

Reply:
Why is it a `TODO`?

Reply:
> Why is it a TODO?
To remind anyone that adds a new profiler

Reply:
A "TODO" marks a task that can be done *now* (and just waits for someone to pick it up?) or alternatively as a reminder for when a specific issue is done. Here it's just a note for the future, unrelated to any specific issue/task. So no need to mark as TODO